lib/v8.js

```diff
@@ -285,14 +294,17 @@ class DefaultSerializer extends Serializer {
    * @returns {void}
    */
   _writeHostObject(abView) {
-    let i = 0;
-    if (abView.constructor === Buffer) {
-      i = bufferConstructorIndex;
-    } else {
-      const tag = ObjectPrototypeToString(abView);
-      i = arrayBufferViewTypeToIndex.get(tag);
-
-      if (i === undefined) {
+    // Keep track of how to handle different ArrayBufferViews. The default
+    // Serializer for Node does not use the V8 methods for serializing those
+    // objects because Node's `Buffer` objects use pooled allocation in many
+    // cases, and their underlying `ArrayBuffer`s would show up in the
+    // serialization. Because a) those may contain sensitive data and the user
+    // may not be aware of that and b) they are often much larger than the
+    // `Buffer` itself, custom serialization is applied.
+    let i = 10;  // FastBuffer
+    if (abView.constructor !== Buffer) {
```

Comment:
I just wonder: checking for subclassing Buffer is not necessary here? As in: using instanceof instead?

Reply:
For the record: I didn't touch this, it's existing behavior. So is the `ObjectPrototypeToString` call, by the way.

To answer your question: I believe the constructor check always works due to Buffer's prototype hacking. Consider the following:
```
> class CustomBuffer extends Buffer {}
undefined
> var b = new CustomBuffer("ok")
undefined
> b
<Buffer 6f 6b>
> b.constructor.name
'Buffer'
> b instanceof CustomBuffer
false
```

Reply:
> For the record: I didn't touch this, it's existing behavior. So is the ObjectPrototypeToString call, by the way.

Yes, I saw that, I was just wondering about it and thought asking here might be better than opening an issue. Thanks for the answer!